### PR TITLE
docs: mention INRIA in the acknowledgment

### PR DESCRIPTION
**Pull request by percevalw**
## Description

Mention INRIA in the acknowledgment.

## Checklist

- [x] If this PR is a bug fix, the bug is documented in the test suite.
- [x] Changes were documented in the changelog (pending section).
- [x] If necessary, changes were made to the documentation (eg new pipeline).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,4 +87,4 @@ If you use EDS-NLP, please cite us as below.
 
 ## Acknowledgement
 
-We would like to thank [Assistance Publique – Hôpitaux de Paris](https://www.aphp.fr/) and [AP-HP Foundation](https://fondationrechercheaphp.fr/) for funding this project.
+We would like to thank [Assistance Publique – Hôpitaux de Paris](https://www.aphp.fr/), [AP-HP Foundation](https://fondationrechercheaphp.fr/) and [INRIA](https://www.inria.fr) for funding this project.
```
Comment (Vincent-Maladiere) [inline README.md:90]: Shall we specify Inria's Soda team? https://team.inria.fr/soda/

Comment (Thomzoy) [reply README.md:90]: And it's `Inria` now apparently :open_mouth:

Comment (percevalw) [reply README.md:90]: > Shall we specify Inria's Soda team? https://team.inria.fr/soda/
@RomainBey ?

Comment (RomainBeyAPHP) [reply README.md:90]: My understanding is that it writes Inria and not INRIA. Last time we talked they were OK for mentioning Inria without specifying the research team.

Comment (percevalw) [inline README.md:90]:
```suggestion
We would like to thank [Assistance Publique – Hôpitaux de Paris](https://www.aphp.fr/), [AP-HP Foundation](https://fondationrechercheaphp.fr/) and [Inria](https://www.inria.fr) for funding this project.
```

**Pull request merged**